setup.py

```diff
@@ -16,22 +16,22 @@
     package_data={'': ['LICENSE']},
     include_package_data=True,
     install_requires=[
-        'cachetools',
-        'cryptography',
-        'fauxfactory',
-        'inflector',
+        'cachetools==1.1.6',
+        'cryptography==1.4',
+        'fauxfactory==2.0.9',
+        'inflection==0.3.1',
```

Comment:
in robottelo we are using inflector, was not able inflection usage (inflection is a requirement   and used by nailgun)